app/src/main/java/com/github/houseorganizer/houseorganizer/MainScreenActivity.java

```diff
@@ -33,9 +34,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         mAuth = FirebaseAuth.getInstance();
 
+        // Setting the sign-out button to switch to the login activity
         findViewById(R.id.sign_out_button).setOnClickListener(v -> {
```

Comment:
This might be better in a method of its own that gets called, like this (with the method signOut(View v) that has the actual code : findViewById(R.id.sign_out_button).setOnClickListener(this::signOut);

Reply:
I did it this way so that when the sign-out button will be moved the person just has to move this code.
It will probably be a good idea to do it in the final place of the button though ^^